### PR TITLE
Added Turkish Language [NEW]

### PR DESCRIPTION
**Pull request by NovaDiablox**
This time done from scratch, can use it comfortably.

### CODE REVIEW

File: garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+
+L.aw_brn3_title = "Kundakçının Ateşi Söner"
+L.aw_brn3_text = "hepsini yaktı, ama şimdi hiç yakıcı el bombası kalmadı! Nasıl başa çıkacaklar!?"
+
+L.aw_fnd1_title = "Adli tabip"
+L.aw_fnd1_text = "etrafta {num} ceset buldu."
+
+L.aw_fnd2_title = "Hepsini Yakalamalıyım"
+L.aw_fnd2_text = "koleksiyonları için {num} ceset buldu.
+
```
Comment (NovaDiablox) [inline garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:1022]: Missing " at the end of the sentence.

Comment (NovaDiablox) [reply garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:1022]: There needs to be a " char after the dot.

Comment (NovaDiablox) [reply garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:1022]: Ok I fixed it.

File: garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua [hunk 2]
```diff
@@ -0,0 +1,1083 @@
...
+-- Credit awards
+L.credit_det_all   = "Dedektifler, performansınız için size {num} ekipman kredisi verildi."
+L.credit_tr_all    = "Hainler, performansınız için size {num} ekipman kredisi verildi."
+
+L.credit_kill = "Bir {role} öldürdüğünüz için {num} kredi aldınız."
+
+-- Karma
+L.karma_dmg_full = "Karman {amount}, bu yüzden bu raunt tam hasar veriyorsun!"
+L.karma_dmg_other = "Karman {amount}. |||UNTRANSLATED_CONTENT_START|||As a result all damage you deal is reduced by {num}%"|||UNTRANSLATED_CONTENT_END|||
```
Comment (svdm) [inline garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:42]: Still problem here

**Review by svdm — CHANGES_REQUESTED** [review]

File: garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua (hunk 2 above)
Comment (NovaDiablox) [reply garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua]: Done

**Pull request merged**